### ISSUE TITLE
[Results] Decoration and hidden elements should be set with aria-hidden or removed

### ISSUE THREAD

**Issue by johanna-skylight** [CDCgov/prime-simplereport#5860]
## Area or Component

Results Page

## Quick Summary
- Buttons download results and clear filters contain icon images used solely are decoration.
- When there are no results listed the pagination nav element is present in the DOM even though it is hidden for sighted users.

### How to Replicate

Load the results page with no results.

## Accessibility Violation

- [The decorative image is not hidden from screen readers](https://docs.deque.com/issue-help/1.0.0/en/alt-text-decorative-inappropriate).
- [Content is not a list but it is marked as such](https://docs.deque.com/issue-help/1.0.0/en/semantic-list-used-for-presentation).

## Action Requested
For the icons set the alt ="", role="presentation" or add an aria-hidden="true".
For the navigation add logic to not include the element in the DOM if there are no results. Second option but not ideal is to add an aria-hidden="true".

<img width="417" alt="Screenshot 2023-05-23 at 10 51 21 AM" src="https://github.com/CDCgov/prime-simplereport/assets/103958711/7b724998-68f4-4200-81c3-0ab876583b04">

<img width="1112" alt="Screenshot 2023-05-23 at 10 53 14 AM" src="https://github.com/CDCgov/prime-simplereport/assets/103958711/661678f3-6cd4-409e-9148-3b85d669f2bd">
